packages/aws-cdk-lib/aws-apigatewayv2/lib/common/stage.ts

```diff
@@ -66,6 +66,13 @@ export interface StageOptions {
    * @default - no throttling configuration
    */
   readonly throttle?: ThrottleSettings;
+
+  /**
+   * The description for the api stage
+   *
+   * @default - none
```

Comment:
nit (Docs from https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-stage.html#cfn-apigatewayv2-stage-description)

```suggestion
   * The description for the API stage
   *
   * @default - no description
```